easybuild/easyblocks/p/pgi.py

```diff
@@ -0,0 +1,94 @@
+##
+# Copyright 2015 Bart Oldeman
```

Comment:
Yes, that works too. If it's licensed under MIT/BSD, we have the freedom of either relicensing it at GPLv2, or to relicense everything 'down' to BSD if we ever wanted to (cfr. https://github.com/hpcugent/easybuild-framework/issues/335), which is the whole point of these requirements.